pkg/util/modules/modules_test.go

```diff
@@ -47,3 +47,109 @@ func TestDependencies(t *testing.T) {
 	assert.Nil(t, svcs)
 	assert.Error(t, err, fmt.Errorf("unrecognised module name: service_unknown"))
 }
+
+func TestRegisterModuleWithOptions(t *testing.T) {
+	sut := NewManager()
+	sut.RegisterModule("module1", mockInitFunc)
+
+	m := sut.modules["module1"]
+
+	assert.NotNil(t, mockInitFunc, m.initFn, "initFn not assigned")
+	assert.True(t, m.public, "module should be public")
+}
+
+func TestRegisterModuleSetsDefaultOption(t *testing.T) {
```

Comment:
This test and `TestRegisterModuleWithOptions` are almost the same. We can remove one of them.

Reply:
Thanks for catching this! I was too focused on refactoring and forgot to re-validate the tests.